sslyze/plugins/http_headers_plugin.py

```diff
@@ -193,14 +227,15 @@ def __init__(
             raw_hsts_header,    # type: Text
             raw_hpkp_header,    # type: Text
             hpkp_report_only,   # type: bool
-            cert_chain          # type: List[cryptography.x509.Certificate]
+            cert_chain,          # type: List[cryptography.x509.Certificate]
+            raw_expect_ct_header # type: Text
```

Comment:
Can you put the `raw_expect_ct_header` argument right after `raw_hpkp_header` (for consistency)?

Reply:
The types are misaligned because of `raw_expect_ct_header`:
<img width="598" alt="screen shot 2018-01-21 at 12 07 02 pm" src="https://user-images.githubusercontent.com/13926056/35198464-222fae46-fea4-11e7-8ed0-e43a13d7f705.png">
 
I think it'd improve readability to add in extra spacing to the rest of the arguments so everything is aligned again: 
<img width="589" alt="screen shot 2018-01-21 at 12 08 13 pm" src="https://user-images.githubusercontent.com/13926056/35198472-4727981c-fea4-11e7-8011-280d90c28148.png">

Not sure how you'd feel about the extra empty spaces! What do you think?

Reply:
Yes that's better 👍